DEV_GUIDE.md

```diff
@@ -165,8 +165,24 @@ To change the container engine to podman set `CONTAINER_ENGINE=podman`
 
 ### Intellij
 
+The project requires JDK-20 to build and run the `integrationtests` module and the IDEA project is configured to build against an SDK
+named `temurin-20`. A suggested way to install this is with [skdman](https://sdkman.io/) using `sdk install java 20-tem`.
+
+Run `mvn clean install -DskipTests` to install the project into your local maven repository (in `~/.m2`). This is necessary because
```

Comment:
should we limit the mvn run to just the krpc-plugin? e.g.
```suggestion
Run `mvn clean install -DskipTests -pl :krpc-code-gen` to install the project into your local maven repository (in `~/.m2`). This is necessary because
```

Reply:
It would also need an `--also-make` but I figured it's worth doing a full build so we don't have to update this command if modules get shuffled around.

Reply:
I don't think we need `--also-make` as `krpc-code-gen` should be stand alone. The only reason to make the switch is we have just moved away from recommending `mvn clean install` to `mvn verify` everywhere else :/

Reply:
Sweet I thought it needed the parent pom installed too, but you're right, no also make required. I do wonder about `verify` though. It doesn't install to ~/.m2 so IDEA barfs on project sync if you've only verified like:
```
Cannot resolve plugin io.kroxylicious:kroxylicious-krpc-plugin:0.3.0-SNAPSHOT

Try to run Maven import with -U flag (force update snapshots)
```

Reply:
Sorry, I wasn't clear. 

We need to run `mvn install -pl :krpc-code-gen` for anything else to work, verify isn't enough in this case. I was trying to make the point that advising a blanket install here goes against the thrust of the the other changes we've been making in the guides.